package.json

```diff
@@ -78,8 +79,8 @@
     "stylelint-webpack-plugin": "^2.1.1",
     "webapp-webpack-plugin": "^2.7.1",
     "webpack": "^4.44.2",
-    "webpack-cli": "^4.1.0",
-    "webpack-merge": "^5.2.0",
+    "webpack-cli": "^3.3.12",
+    "webpack-merge": "^4.2.2",
```

Comment:
I had to downgrade a couple of dependencies after a few students reported issues.